### PR TITLE
github action to replace circleci, borrowed from my earlier work on apostrophe-static-i18n

### PR DESCRIPTION
**Pull request by boutell**
We're using a third party action for mongodb support, not that it would be very hard to do our own using "m", but it's there.

Works great, currently picks up an incompatibility that's breaking our unit tests in node 16.

### CODE REVIEW

**Comment by boutell** [comment]
I guess we don't merge this until that other bug is fixed, but it's a real bug this PR enabled us to detect.

File: .github/workflows/main.yml
Comment (abea) [inline .github/workflows/main.yml:22]: ❤️

**Review by abea — COMMENTED** [review]
It looks great. It's a little hard to fully ✅ before the tests are passing.

**Comment by boutell** [comment]
@abea this now passes everything except circleci which fails for reasons external to the PR (github is still configured to try to run it, and it doesn't find its folder, which, well, it shouldn't after this PR is merged). Pls review. Thanks!

**Review by abea — APPROVED** [review]

**Pull request merged**
